ldk/javascript/examples/self-test-loop/src/tests/network/index.ts

```diff
@@ -155,25 +143,25 @@ export const testWebsocketConnection = (): Promise<boolean> =>
 
           clipboardListener.cancel();
           console.info('Clipboard listener cancelled');
-
           cancellable.cancel();
         });
+
+        await socket.setCloseHandler((error, code, text) => {
```

Comment:
Im wondering if we should do this before we do `const cancellable = await socket.setMessageHandler` I'm not sure if setCloseHandler is actually being called since were awaiting the setMessageHandler and the first thing were doing is writing a message

Reply:
Im also wondering if the very first thing we should do is call `setMessageHandler`

Reply:
If I put setCloseHandler before all the other handlers, it will stuck in setCloseHandler, since we are not calling socket close yet. 
I have tested switch setMessageHandler and WriteMessage doesn't affect the test result. I would move setMessageHandler be the first thing to call.